src/event-handler.js

```diff
@@ -16,32 +16,14 @@ A non-ecommerce event has the following schema:
 function EventHandler(common) {
     this.common = common || {};
 }
-EventHandler.prototype.logEvent = function(event) {
-    var ignoredEvents = [
-        'click',
-        'change',
-        'submit'
-    ];
+EventHandler.prototype.logEvent = function (event) {
+    var ignoredEvents = ['click', 'change', 'submit'];
 
     if (ignoredEvents.includes(event.EventName.toLowerCase())) {
         return;
     }
 
     window.heap.track(event.EventName, event.EventAttributes);
 };
-EventHandler.prototype.logError = function(event) {
-    // The schema for a logError event is the same, but noteworthy differences are as follows:
-    // {
-    //     EventAttributes: {m: 'name of error passed into MP', s: "Error", t: 'stack trace in string form if applicable'},
-    //     EventName: "Error"
-    // }
-};
-EventHandler.prototype.logPageView = function(event) {
```

Comment:
i believe this will cause an error in the wrapper.  https://github.com/mparticle-integrations/mparticle-javascript-web-kit-wrapper/blob/master/index.js#L160-L190

This doesn't check for the existence before calling.

So @SbDove - there are no page views in heap?